### PR TITLE
fix: Clarify rebuild home persistence [#46]

### PR DESCRIPTION
**Pull request by vapurrmaid**
Resolves: #46

### CODE REVIEW

File: environments/lifecycle.md [hunk 1]
```diff
@@ -12,10 +12,9 @@ on cloud compute and justify more powerful dev machines :).
 Rebuilding an [Environment](index.md) allows you to update to the latest image,
 edit resource requests, or restart your environment after a shutdown.
 
-Only the `/home` directory is persisted between rebuilds. This allows the
-Image to populated system development dependencies without interfering with any
-source code or miscellaneous configuration unique to your individual development
-environment.
+Only the `/home` directory is persisted between rebuilds. Configurations and source
```
Comment (cmoog) [inline environments/lifecycle.md:15]:
```suggestion
Only the `/home/<username>` directory is persisted between rebuilds. Configurations and source
```

File: environments/lifecycle.md [hunk 2]
```diff
@@ -12,10 +12,9 @@ on cloud compute and justify more powerful dev machines :).
...
+code within the `/home` subtree will not be affected across rebuilds, even if the
```
Comment (cmoog) [inline environments/lifecycle.md:16]:
```suggestion
code within the `/home/<username>` subtree will not be affected across rebuilds, even if the
```

File: environments/lifecycle.md [hunk 3]
```diff
@@ -12,10 +12,10 @@ on cloud compute and justify more powerful dev machines :).
...
 edit resource requests, or restart your environment after a shutdown.
 
-Only the `/home` directory is persisted between rebuilds. This allows the
-Image to populated system development dependencies without interfering with any
-source code or miscellaneous configuration unique to your individual development
-environment.
+Only the `/home/<username>` directory is persisted between rebuilds.
+Configurations and source code within the `/home/<username>` subtree will
+not be affected across rebuilds, even if the underlying
+[Image](../images/index.md) or its dependencies change.
```
Comment (khorne3) [inline environments/lifecycle.md:18]: Maybe

> Only the `/home/<username>` directory persists between rebuilds. Rebuilds do not affect configurations and source code within the `/home/<username>` subtree, even if the underlying [Image](../images/index.md) or its dependencies change.

?

Comment (cmoog) [reply environments/lifecycle.md:18]: ah much better voice 👍

**Review by khorne3 — APPROVED** [review]

**Pull request merged**